<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WholeImageForReviewPropertyClass</class>
 <widget class="QDialog" name="WholeImageForReviewPropertyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WholeImageForReviewForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbRotation">
     <property name="title">
      <string>Image Rotation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Front</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbFrontRotation">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>90</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>180</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>270</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Back</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbBackRotation">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>90</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>180</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>270</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QGroupBox" name="gbFrontBackOrientation">
       <property name="title">
        <string>FrontBackOrientation</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="rbFrontBackHoriaontal">
          <property name="text">
           <string>Horiaontal</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rbFrontBackVertical">
          <property name="text">
           <string>Vertical</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="gbPhaseOrientation">
       <property name="title">
        <string>PhaseImageOrientation</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="rbPhaseHoriaontal">
          <property name="text">
           <string>Horiaontal</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rbPhaseVertical">
          <property name="text">
           <string>Vertical</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QCheckBox" name="cbNGNailCrossView">
       <property name="text">
        <string>NGNailCrossView</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="cbNGPointView">
       <property name="text">
        <string>NGPointView</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="cbOnlyCurrentNailView">
       <property name="text">
        <string>OnlyCurrentNailView</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="WholeImageForReview.qrc">
         <normaloff>:/Qtres/flash_swf_128.PNG</normaloff>:/Qtres/flash_swf_128.PNG</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="WholeImageForReview.qrc">
         <normaloff>:/Qtres/flash_64.PNG</normaloff>:/Qtres/flash_64.PNG</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbFrontBackHoriaontal</tabstop>
  <tabstop>rbFrontBackVertical</tabstop>
  <tabstop>rbPhaseHoriaontal</tabstop>
  <tabstop>rbPhaseVertical</tabstop>
  <tabstop>cbNGNailCrossView</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources>
  <include location="WholeImageForReview.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbOk</sender>
   <signal>clicked()</signal>
   <receiver>WholeImageForReviewPropertyClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>WholeImageForReviewPropertyClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
